### ISSUE TITLE
Custom Slide Animation Duration

### ISSUE THREAD

**Issue by Josephjbliss** [xiaolin/react-image-gallery#129]
Currently, to adjust the actual animation duration of the slide, I need to add something like:

```css
.image-gallery-slide {
 transition-duration: .8s !important;
}
```

in order to override the JS-injected inline CSS. It would be awesome to have this as a prop to the component.

**Comment by xiaolin** [comment]
Sure, I will add this soon.

**Comment by Josephjbliss** [comment]
Awesome. Thank you! 💯

**Comment by xiaolin** [comment]
released in v0.7.10
